playwright.config.js

```diff
@@ -38,15 +38,15 @@ module.exports = defineConfig({
 
   /* Configure projects for major browsers */
   projects: [
-    {
-      name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
-    },
+    // {  //disabled as test case failing on multiple browsers
+    //   name: 'chromium',
+    //   use: { ...devices['Desktop Chrome'] },
+    // },
```

Comment:
is it intentional?
If yes, do you think we need this in future or can be removed?

Let's add a comment in case we have a plan to un comment this in future in what scenarios.

Reply:
@Satendra-SR yes it is intentional since when test case is executed on multiple browser, the new volunteer  details volunteer  based on their status (New signups, Active and lead volunteer) are not available on the 1st page, hence  need to execute the test case in single browser